### DISCUSSION TITLE
How can I find a solution to enter only letters and spaces in input?

### DISCUSSION THREAD

**Discussion by olcaneristi** [react-hook-form/react-hook-form#4241] [Q&A] [answered]
I want to input only character (Turkish characters included) and space key in an input prepared with react-hook-form. I want numbers and special characters not to be written to input when pressed. 

I provided this as a warning with regex, but I couldn't block character input.

![image](https://user-images.githubusercontent.com/45832621/108474660-b9a71200-72a0-11eb-9797-487467c30882.png)

Thanks!

**Comment by bluebill1049** [comment, marked as answer]
i think you should use a Mask input lib, google around there are plenty of them.

**Reply by olcaneristi** [reply]
thanks for your answer. i fixed it. the function has both character control and capitalizes text.

![image](https://user-images.githubusercontent.com/45832621/109499543-188d3800-7aa6-11eb-9592-a334ba4dbab2.png)
